kuma/api/v1/tests/test_views.py

```diff
@@ -321,3 +324,53 @@ def test_whoami(user_client, api_settings, wiki_user, beta_testers_group,
         }
     }
     assert_no_cache_header(response)
+
+
+# NOTE(peterbe) These are manually cloned from the pytest fixture in
+# kuma.api.conftest. The reason why is because we have to use the
+# ElasticTestCase class for Elasticsearch fixtures. So we can't use the
+# regular pytest fixtures and that's why we have to replicate these tricks.
+@override_settings(
+    BETA_HOST='beta.mdn.dev',
+    ALLOWED_HOSTS=settings.ALLOWED_HOSTS + [settings.BETA_HOST],
+    ENABLE_RESTRICTIONS_BY_HOST=True)
+class APISearchIntegration(ElasticTestCase):
+
+    fixtures = ElasticTestCase.fixtures + ['wiki/documents.json']
+
+    def test_search_en_US(self):
+        url = reverse('api.v1.search', args=['en-US'])
+        response = self.client.get(url, HTTP_HOST=settings.BETA_HOST)
+        assert response.status_code == 400
+        assert response['content-type'] == 'application/json'
+        assert response.json() == {
+            'error': "No query string 'q'"
+        }
+
+        response = self.client.get(
```

Comment:
Nit: I'd lean toward making each of these sections a separate test method, but feel free to ignore if you like it as is.

Reply:
Good point. I think I did it like that to over-optimize. It makes it faster by not having to have to do the expensive index set up and tear down and the fixture loading.

Reply:
I was pondering it and decided it's not worth it. The setup/teardown is more expensive than the value in doing the tests "properly".